package.json

```diff
@@ -29,7 +29,7 @@
     "gatsby-plugin-offline": "^3.1.2",
     "gatsby-plugin-react-helmet": "^3.2.4",
     "gatsby-plugin-sass": "2.3.1",
-    "gatsby-plugin-sharp": "^2.5.7",
+    "gatsby-plugin-sharp": "2.6.36",
```

Comment:
Is there a reason to pin to a specific version? Not sure why the Sass one above is either

Reply:
I'm not sure why it ended up like that. I just did `npm audit fix` and thats what the result was. Should I change it to `^2.6.36`

Reply:
OK, if the audit is changing it automatically I don't think it needs to be changed here. Might be good to look at the others and fix them all separately